src/main/resources/static/myProfile.html

```diff
@@ -0,0 +1,178 @@
+
```

Comment:
Page doesn't actually work. Changes have already been made to this page to fix the functionality.